### PR TITLE
Note/emotion tracking elderly

### PR DESCRIPTION
**Pull request by Panoj-k**
Add emotion tracking card
image

### CODE REVIEW

File: src/components/organisms/ElderlyCard.tsx [hunk 1]
```diff
@@ -8,6 +8,8 @@ import { useTranslation } from 'react-i18next';
 import useDimensions from '../../hooks/useDimensions';
 import FallbackImage from '../molecules/FallbackImage';
 
+// const ProfilePic = require('../../assets/images/profile.png');
```
Comment (chuwong35122) [inline src/components/organisms/ElderlyCard.tsx]: consider removing this comment

File: src/components/organisms/EmotionCard.tsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+          </View>
+        </Modal.Footer>
+      </Modal.Content>
+    </Modal>
+  );
+};
+
+export default EmotionCard;
+
+const styles = StyleSheet.create({
```
Comment (chuwong35122) [inline src/components/organisms/EmotionCard.tsx:113]: You could write an in-line styling if you use native base.
For instance <View alignItems='center' ...>
(optional)

File: src/components/organisms/EmotionCard.tsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  };
+  const sendPayload = async (emotion) => {
+    const payload = {
+      emotionLevel: emotion
+    };
+    try {
+      const { data } = await client.post('/emotion-tracking', payload);
+      if (data) console.log('emotion sent: ' + emotion);
+    } catch (err) {
+      console.log('Unsuccessful emotion sent: ' + emotion);
```
Comment (chuwong35122) [inline src/components/organisms/EmotionCard.tsx]: remove console.log

File: src/screens/ElderlyEmotionHistory.tsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  const { width, height } = useWindowDimensions();
+  const [hist, setHist] = useState<EmotionalHistory[]>([]);
+
+  const toast = useToast();
+  function handleDayPress(val: EmotionalHistoryFrequency) {
+    toast.show({ title: JSON.stringify(val) });
+  }
+
+  useEffect(() => {
+    const data: EmotionalHistory[] = [
```
Comment (chuwong35122) [inline src/screens/ElderlyEmotionHistory.tsx]: This is the old version of mine before connecting to back-end, can you change it to the newest version?

**Review by chuwong35122 — CHANGES_REQUESTED** [review]

File: src/screens/ElderlyHomeScreen.tsx [hunk 5]
```diff
@@ -56,9 +73,38 @@ const ElderlyHomeScreen = () => {
...
+  useAsyncEffect(async () => {
+    const emotionDate = moment(await getEmotionDate());
+    const todayDate = moment().format('L');
+    if (!user) {
+      return;
+    }
+    const uid = user?.uid;
+    const emotionTracking = await getIsEmotionTrackingOn(uid);
+    const isEmotionTrackingOn = emotionTracking.isEnabled;
+    console.log(isEmotionTrackingOn);
```
Comment (chuwong35122) [inline src/screens/ElderlyHomeScreen.tsx]: oops, can you remove this too?

Comment (Panoj-k) [reply src/screens/ElderlyHomeScreen.tsx]: Oh, I forgot.

**Review by chuwong35122 — APPROVED** [review]

**Pull request merged**